CI/Dockerfile

```diff
@@ -80,6 +80,8 @@ RUN /root/etc/CI/docker/build_hdf5.sh
 
 FROM hdf5 AS moab
 
+ENV PYTHONPATH=/root/build_dir/moab/bld/pymoab/lib/python3.6/site-packages
```

Comment:
This makes me wonder if we are installing MOAB in the wrong place?  Or not installing it at all?  Or we are installing it in the right place and we should reference *that* place here instead?

Reply:
we could add a python import command to the dockerfile to see if it is found by python if that helps check it it was installed

```bash
python -c "import pymoab"
```

Reply:
The problem does not comes from us.
It comes from Moab.
when they want to test pymoab they install it with `pip -e` for testing purposes
The problem is this is prevented because `/root/build_dir/moab/bld/pymoab/lib/python3.6/site-packages` is not in our PYTHONPATH

Reply:
Hmmm… maybe the problem is that line 91 in the pymoab CMake is commented out?!?

Reply:
Looks like @pshriwise commented this out in 2018 🤣